src/extra/components/pulses.py

```diff
@@ -36,32 +36,50 @@ class PulsePattern:
     pattern table.
     """
 
-    # An implementation of this base class is required to implement the
-    # _get_pulse_mask() method.
+    # All methods are built on top of get_pulse_mask and trains()
+    # The default implementations use _mask_table() and _get_ppdecoder_node()
 
-    # Regular expressions for timeserver control and pipeline data.
+    # Number of elements in bunch pattern table.
+    _bunch_pattern_table_len = 3611
```

Comment:
I'm probably being a little paranoid here, but could the comment link to the docs/redmine ticket where this is stated? :sweat_smile: To me it's a bit of a random number.

Reply:
Fixed in d7a5c9e. Note that this document is only available internally in Alfresco.